app/src/main/java/org/piramalswasthya/sakhi/helpers/Konstants.kt

```diff
@@ -26,9 +26,9 @@ object Konstants {
     const val minAgeForNcd: Int = 30
     const val minAgeForReproductiveAge: Int = 15
     const val maxAgeForReproductiveAge: Int = 49
-    const val maxAgeForInfant: Int = 1
-    const val minAgeForChild: Int = 2
-    const val maxAgeForChild: Int = 5
+    const val maxAgeForInfant: Int = 42
```

Comment:
These validations are off.

Reply:
Yes, in another PR it has changed.
Checked with @ritik-bizbrolly, validations were changed twice.
cc: @madhavaramu83